customer-docs/user-manual.md

```diff
@@ -451,4 +452,8 @@ CLI Arguments:
 | template-directory | String | True | Location of the templates |
 
 CMD : `<install directory>/bin/streamliner scripts --config output/<pipeline name>/<environment>/conf/streamliner-configuration.yml --type-mapping <install-directory>/conf/type-mapping.yml  --template-directory <install directory>/templates/<snowflake | hadoop>`
- 
+
+## Migrating Templates from Streamliner 4.x to 5.0
+1. Replace the imports and variables to use new Java POJO classes.
```

Comment:
Can you provide an example for each item on this list? Maybe a few examples for the first one.